tdrs-backend/tdpservice/settings/common.py

```diff
@@ -261,6 +230,13 @@ class Common(Configuration):
     # Custom user app
     AUTH_USER_MODEL = "users.User"
 
+    # Username or email for initial Django Super User
+    # NOTE: In a deployed context this will default to the Product Owner
+    DJANGO_SUPERUSER_NAME = get_required_env_var_setting(
```

Comment:
This setting is used in a migration, catching the error here prevents possible issues that could arise from that migration failing.